rust/tvm-sys/src/datatype.rs

```diff
@@ -83,6 +83,10 @@ impl DataType {
         DataType::new(DL_FLOAT_CODE, bits, lanes)
     }
 
+    pub const fn float32() -> DataType {
```

Comment:
if this is `float32`, should the old `float` be renamed to `float64` (to avoid clashing with `float` vs. `double` size intuition?)

Reply:
We don't use float/double terminology in TVM due to the fact that both integers and floating point values are both parametric by vector length and bit width. The float method here takes both bit width and vector length, unless I'm missing something.